src/modules/trash/trash.usecase.ts

```diff
@@ -7,6 +7,9 @@ import { FileUseCases } from '../file/file.usecase';
 
 @Injectable()
 export class TrashUseCases {
+  async clearTrash(user: User) {
```

Comment:
The same as in previous cases, this method is raised within the tests, I would understand that TDD is being applied and we must implement it later, I leave the model explicitly in the code.